core/templates/domain/exploration/LostChangeObjectFactory.ts

```diff
@@ -150,8 +160,8 @@ export class LostChange {
     if ((<LostChangeValues> this.newValue).feedback &&
     (<LostChangeValues> this.oldValue).feedback) {
```

Comment:
can we firstly retype these values into new variables and only then use them in the ifs.

Reply:
done